test/instrumentation/modules/tedious.test.js

```diff
@@ -48,7 +48,7 @@ function withConnection (t) {
     const conn = new tedious.Connection(connOpts)
     const onConnect = (err) => {
       if (err) return reject(err)
-      resolve(conn)
+      return resolve(conn)
```

Comment:
Did this make a difference anywhere? From looking at the tedious source code, I don't think it should.